.github/workflows/ci.yml

```diff
@@ -125,7 +125,7 @@ jobs:
       - name: Setup python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.6
+          python-version: 3.8
```

Comment:
Highly likely that v3.6 on the Github Actions build machine is just broken, which could explain the previously failing tests.